integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -516,4 +476,25 @@ class ApplicationsIT extends ITSupport {
         AppUser readAppUser = app.getApplicationUser(appUser2.getId())
         assertThat readAppUser.getCredentials().getUserName(), equalTo("updated-"+user2.getProfile().getEmail())
     }
+
+    @Test
+    void testOAuth2Flow() {
+
+        Client client = Clients.builder()
+            .setOrgUrl("https://java-sdk.oktapreview.com")
+            .setCacheManager(new DisabledCacheManager())
+            .setAuthenticationScheme(AuthenticationScheme.OAUTH2)
+            .setAuthorizationMode("PrivateKey")
+            .setScopes(Arrays.asList("okta.apps.read", "okta.apps.manage"))
+            .setKeyFilePath(getClass().getResource("/privateKey.pem").file)
```

Comment:
ITs will run against a real org. Having a hardcoded private key checked is fine but we should provide the ability to override this config

Reply:
Ideally, just generate one with the Pem writer from BC, you can test with a RS* and ES* keys this way too)

Reply:
Noted. Unit tests pushed do it this way. Will add the same to the upcoming ITs.